### ISSUE TITLE
Add an address book section

### ISSUE THREAD

**Issue by Darlington02** [horuslabsio/Starknet-Scaffold#83]
Simply replicate [this section in Starkscan](https://starkscan.co/address-book), but this time listing all relevant contract addresses within the ecosystem.

TIP: 
you can simply reuse the tables in the Wikipedia section.

**Comment by vibenedict** [comment]
v

**Comment by Jemiiah** [comment]
Ans

**Comment by Gift-Naomi** [comment]
Can i

**Comment by martinvibes** [comment]
i

**Comment by petersdt** [comment]
@Darlington02 can i take on this

**Comment by juandiegocv27** [comment]
Hi, @Darlington02 @vibenedict Hi, @Darlington02,

I would love to contribute to the task "Add an address book section #83" starting on May 23, 2024, with a completion date of June 1, 2024. I'm Juan Diego Carballo, a full-stack developer with a software engineering degree. I have already contributed to other projects and am currently creating an on-chain game. I have experience with Cairo and blockchain development, including implementing and optimizing encoding algorithms. I have already joined the Telegram group for contributors.

To complete this issue, I will:
1. Review the existing section in Starkscan for reference.
2. Replicate the section, listing all relevant contract addresses within the ecosystem.
3. Reuse the tables from the Wikipedia section to ensure consistency and efficiency.
4. Test the new section to ensure it integrates seamlessly with the existing codebase.
